app/views/alchemy/admin/pages/_form.html.erb

```diff
@@ -18,7 +18,7 @@
 
   <%= f.input :name, autofocus: true %>
   <%= f.input :urlname, as: 'string', input_html: {value: @page.slug}, label: Alchemy::Page.human_attribute_name(:slug) %>
-  <alchemy-char-counter data-count="60">
+  <alchemy-char-counter count="60">
```

Comment:
I would prefer `max-chars` or simply `max` as the prop name.

Reply:
I changed the attribute to `max-chars`.